packages/big-design/src/components/Button/styled.tsx

```diff
@@ -58,13 +58,14 @@ export const StyledButton = styled.button<ButtonProps & MarginProps>`
 
   ${({ theme }) => theme.breakpoints.tablet} {
     width: auto;
-  }
 
-  ${({ iconOnly: icon, theme }) =>
-    icon &&
-    css`
-      padding: 0 ${theme.spacing.xSmall};
-    `};
+    ${({ iconOnly: icon, theme }) =>
+      icon &&
+      css`
+        padding: 0;
+        width: ${addValues(theme.spacing.xxLarge, theme.spacing.xxSmall)};
```

Comment:
can we use `min-width` here? `min-width` should help if an icon will be wider then expected.

Reply:
I guess, but the height is fixed, so it still will be able to overflow the button.

Reply:
oh, I check the existing code, we need to increase the horizontal padding instead of adding `width`.

Reply:
Setting vertical padding to 5px/0.3125rem is an option